itl/command-plugins.conf

```diff
@@ -1696,6 +1696,7 @@ object CheckCommand "by_ssh" {
 	}
 
 	vars.by_ssh_address = "$check_address$"
+	vars.by_ssh_port = "$ssh_port$"
```

Comment:
`ssh_port` isn't defined anywhere, so the user must do that on the host level. Isn't it more reasonable to just define `by_ssh_port` if needed? check_by_ssh assumes the default 22 if you omit its value.